lib/Client.php

```diff
@@ -203,4 +305,42 @@ private function message($msg)
 
         return $msg;
     }
+
+    private function loadFeatureFlags(): void
+    {
+        $response = $this->httpClient->sendRequest(
+            '/api/feature_flag',
+            null,
+            [
+                "Authorization: Bearer $this->personalApiKey",
+            ]
+        );
+        if (401 === $response->getResponseCode()) {
+            throw new Exception(
+                "Your personalApiKey is invalid. Are you sure you're not using your Project API key?
+                 More information: https://posthog.com/docs/api/overview"
+            );
+        }
+
+        $responseBody = json_decode($response->getResponse(), true);
+        if (null === $responseBody) {
+            return;
+        }
+
+        if (empty($responseBody)) {
+            $this->featureFlags = [];
+        }
+
+        $this->featureFlags = $responseBody['results'];
```

Comment:
Another change we made is that this should filter by active flags.

The `active` property is in `$responseBody['results'][i]['active']`